test/mocks.go

```diff
@@ -1,44 +0,0 @@
-// Copyright 2015 ISRG.  All rights reserved
```

Comment:
I disagree with Mr. jmhodges here; I think pulling this out of test was a good move, and in line with this PR's goals.